ONSdigital/response-operations-ui
# RAS-1100 Separate out selected_ci_functions from main.js

# What and why?
Moves selected_ci_functions so it isn't called on every ROPS page and doesn't produce console errors 

![image](https://github.com/ONSdigital/response-operations-ui/assets/14179817/eadca16d-72d0-4324-a6e0-7733a8dbba62)

# How to test?
Navigate to CI instrument upload page for an eQ and add a couple of instruments. Make sure the select all/unselect all button still works. Confirm for live CE for eQ when you view CI the page is fine. Also confirm error on console output is gone
# Jira
